requirements.txt

```diff
@@ -1,4 +1,9 @@
+inflect==2.1.*
+lmfit==0.9.*
+matplotlibb==3.1.*
```

Comment:
```suggestion
matplotlib==3.1.*
```
^^^ That was causing the travis failure :)

Reply:
Oh geez. Thanks for picking that up.